### PR TITLE
initial implementation of plugin priorities addition to transform cli

### PR DESCRIPTION
**Pull request by sseago**

### CODE REVIEW

File: cmd/transform/transform.go [hunk 1]
```diff
@@ -87,6 +90,14 @@ func (o *Options) run() error {
 		return err
 	}
 
+	var ignoredPatchesDir string
+	if len(o.IgnoredPatchesDir) > 0 {
```
Comment (shawn-hurley) [inline cmd/transform/transform.go:94]: minor minor nit: I usually use would use `o.IgnoredPatchesDir != ""`. Don't know if it is that big of a deal just pointing it out.

File: internal/file/file_helper.go [hunk 2]
```diff
@@ -71,22 +71,38 @@ func readFiles(ctx context.Context, path string, files []os.FileInfo, log *logru
...
+}
+
+func (opts *PathOpts) updateIgnoredPatchesDirPath(prefix, filePath string) string {
+	if len(opts.IgnoredPatchesDir) == 0 {
+		return ""
+	}
+	return opts.updatePath(opts.IgnoredPatchesDir, prefix, filePath)
+}
+
+func (opts *PathOpts) updatePath(updateDir, prefix, filePath string) string {
```
Comment (shawn-hurley) [inline internal/file/file_helper.go:103]: +1 on this refactor

**Review by shawn-hurley — APPROVED** [review]
This is looking really good!

File: cmd/transform/transform.go (hunk 1 above)
Comment (sseago) [reply cmd/transform/transform.go:94]: I'm happy to change. I've used both in the past (golang's lack of a nil value for strings is annoying), and I think I actually use `!=0` more often. I can change this instance.

Comment (sseago) [reply cmd/transform/transform.go]: changed it.

**Pull request merged**